storage/Devices/EncryptionImpl.cc

```diff
@@ -196,6 +196,52 @@ namespace storage
 
 
     Text
+    Encryption::Impl::do_resize_text(ResizeMode resize_mode, const Device* lhs, Tense tense) const
+    {
+	const Encryption* encryption_lhs = to_encryption(lhs);
+
+	Text text;
+
+	switch (resize_mode)
+	{
+	    case ResizeMode::SHRINK:
+		text = tenser(tense,
+			      // TRANSLATORS: displayed before action,
+			      // %1$s is replaced by partition name (e.g. /dev/sda1),
+			      // %2$s is replaced by old size (e.g. 2GiB),
+			      // %3$s is replaced by new size (e.g. 1GiB)
+			      _("Shrink encryption on %1$s from %2$s to %3$s"),
```

Comment:
As I [said before](https://github.com/openSUSE/libstorage-ng/pull/142#r78337392), this does not make sense in English. Encryption cannot be shrunk. An encryption layer, or an encrypted device, can.